pkg/server/app/errors.go

```diff
@@ -0,0 +1,11 @@
+package app
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	ErrAppAlreadyExists = status.Errorf(codes.AlreadyExists, "App already exists")
+	ErrAppNotFound      = status.Errorf(codes.NotFound, "App not found")
```

Comment:
maybe the name `app.ErrNotFound` is good enough. (I mean, you're already on `app` package)